frontend/public/components/factory/modal.tsx

```diff
@@ -67,7 +67,7 @@ export const ModalTitle: React.SFC<ModalTitleProps> = ({
   className = 'modal-header',
 }) => (
   <div className={className}>
-    <h4 className="modal-title">{children}</h4>
+    <h1 className="pf-c-title pf-m-2xl">{children}</h1>
```

Comment:
This change is going to break:
* https://github.com/openshift/console/blob/master/frontend/packages/kubevirt-plugin/integration-tests/tests/models/editFlavorView.ts#L5
* https://github.com/openshift/console/blob/master/frontend/packages/kubevirt-plugin/integration-tests/views/editCDView.ts#L14

Reply:
updated tests